data-model/SDP-sample-records/codicological-unit_sample1.json

```diff
@@ -0,0 +1,143 @@
+{
+  "id": "ark:/21198/a310jfBd",
+  "type": "Codicological Unit",
+  "label": "Part 1",
+  "extent": ["f. 1-152"],
```

Comment:
I know it's possible to find different methods for numbering can exist in the same ms (e.g., folio number with page number). Regardless of if the system is mixed or not, I wonder if we want to have a place to record what system is used, along with a note field for more detailed description of oddities? Could this go in extent? It would be necessary to tell the user if the foliation behaves oddly at any point, especially if they want to use this data in any machine-assisted way.

Reply:
Maybe the regularity of the string, including the use of 'f.' and 'p.' would cover all situations? But it's worth thinking on further.

Reply:
Extent will become like we use in ms object: 153 ff.

We will add a foliation field (here and in ms object?). Can be as complex as needed for the object/part/text

Reply:
Foliation added. Can use "f."/"ff." for folio(s); "p."/"pp." for page(s), if paginated.